docs/api/python/autograd/autograd.md

```diff
@@ -76,7 +82,63 @@ Detailed tutorials are available in Part 1 of
 [the MXNet gluon book](http://gluon.mxnet.io/).
 
 
+# Higher order gradient
+
+Some operators support higher order gradients. Meaning that you calculate the gradient of the
+gradient. For this the operator's backward must be differentiable as well. Some operators support
```

Comment:
"For this the operator's backward must be differentiable as well".  This sentence also seems not accurate and necessary.

Reply:
I can remove it, how would you put it in a more accurate way?